gulpfile.js

```diff
@@ -0,0 +1,42 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var rename = require('gulp-rename');
+var cleanCSS = require('gulp-clean-css');
+var autoprefixer = require('gulp-autoprefixer');
+var browserSync = require('browser-sync').create();
+
+gulp.task('default', ['serve']);
+
+// Static Server + watching sass/html files
+gulp.task('serve', ['minify-css'], function() {
+
+    browserSync.init({
```

Comment:
The rest of the files in the repo use two spaces for indentation, this should be changed to conform to that